common4j/src/main/com/microsoft/identity/common/java/commands/parameters/nativeauth/AcquireTokenNoFixedScopesCommandParameters.java

```diff
@@ -0,0 +1,147 @@
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+package com.microsoft.identity.common.java.commands.parameters.nativeauth;
+
+import com.google.gson.annotations.Expose;
+import com.microsoft.identity.common.java.authscheme.AbstractAuthenticationScheme;
+import com.microsoft.identity.common.java.dto.IAccountRecord;
+import com.microsoft.identity.common.java.exception.ArgumentException;
+import com.microsoft.identity.common.java.exception.ClientException;
+import com.microsoft.identity.common.java.exception.TerminalException;
+import com.microsoft.identity.common.java.logging.Logger;
+import com.microsoft.identity.common.java.providers.microsoft.azureactivedirectory.AzureActiveDirectory;
+import com.microsoft.identity.common.java.providers.microsoft.azureactivedirectory.AzureActiveDirectoryCloud;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.util.List;
+import java.util.Map;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.experimental.SuperBuilder;
+
+/**
+ * AcquireTokenNoFixedScopesCommandParameters defines the parameters used for
+ * [AcquireTokenNoFixedScopesCommand] class.
+ */
+@Getter
+@EqualsAndHashCode(callSuper = true)
+@SuperBuilder(toBuilder = true)
+public class AcquireTokenNoFixedScopesCommandParameters extends BaseNativeAuthCommandParameters {
+
+    private static final String TAG = AcquireTokenNoFixedScopesCommandParameters.class.getSimpleName();
+
+    private static final Object sLock = new Object();
+
+    private final IAccountRecord account;
+
+    @Expose()
+    @NonNull
+    private final AbstractAuthenticationScheme authenticationScheme;
+
+    @Expose()
+    private final boolean forceRefresh;
+
+    private final String loginHint;
+
+    private final List<Map.Entry<String, String>> extraOptions;
+
+    /**
+     * Validates the command parameters in this object are consistent and can be used for
+     * command execution.
+     * @throws ArgumentException
+     */
+    public void validate() throws ArgumentException {
+        final String methodName = ":validate";
+
+        Logger.verbose(
+                TAG + methodName,
+                "Validating operation params..."
+        );
+
+        // AuthenticationScheme is present...
+        if (null == authenticationScheme) {
+            throw new ArgumentException(
+                    ArgumentException.ACQUIRE_TOKEN_SILENT_OPERATION_NAME, // TODO update argument name
```

Comment:
This TODO still relevant?

Reply:
@SaurabhMSFT yeah I think we should update it - the argument name isn't in line with the command type.

Reply:
I have changed it